helm-chart/binderhub/templates/NOTES.txt

```diff
@@ -1,3 +1,148 @@
+{{- define "removedConfig" }}
+
+It looks like you still have some unsupported configuration!
+
+The binderhub chart has removed special-handling
+of many simple configurables in the BinderHub source code.
+Instead, the Python traits configuration is exposed directly.
+
+See https://binderhub.readthedocs.io/en/latest/reference/ref-index.html
+for what can be configured.
+
+In general, it works like:
+
+config:
+  ClassName:
+    trait_name: value
+
+e.g. to set config on BinderHub, e.g., use:
+
+config:
+  BinderHub:
+    use_registry: false
+
+or on GitHubRepoProvider:
+
+config:
+  GitHubRepoProvider:
+    access_token: "..."
+
+
```

Comment:
In these examples, where `use_registry` and `access_token` are traits to be configured.